collector/megacli_test.go

```diff
@@ -48,7 +51,32 @@ func TestMegaCliDisks(t *testing.T) {
 		t.Fatalf("Unexpected drive temperature: %s", stats[32][0]["Drive Temperature"])
 	}
 
+	if stats[32][1]["Drive Temperature"] != "N/A" {
+		t.Fatalf("Unexpected drive temperature: %s", stats[32][2]["Drive Temperature"])
+	}
+
 	if stats[32][3]["Predictive Failure Count"] != "23" {
 		t.Fatal()
 	}
 }
+
+func TestMegaCliCollector(t *testing.T) {
```

Comment:
Since this is not a general test of the collector, but a specific regression test that just checks that it doesn't crash, how about calling it `TestMegaCliCollectorDoesntCrash`?